maintainers/maintainer-list.nix

```diff
@@ -17163,6 +17163,12 @@
     githubId = 449990;
     name = "Cedric Cellier";
   };
+  rixxc = {
```

Comment:
Please add yourself to maintainer list in separate commit `maintainers: add rixxc`